pkgs/by-name/wr/wrangler/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  fetchFromGitHub,
+  pnpm_9,
+  nodejs,
+  npmHooks,
+  writeScriptBin
+}:
+let
+  srcHash = "sha256-/4iIkvSn85fkRggmIha2kRlW0MEwvzy0ZAmIb8+LpZQ=";
+  pnpmDepsHash = "sha256-aTTaiGXm1WYwmy+ljUC9yO3qtvN20SA+24T83dWYrI0=";
+in
```

Comment:
There is no real reason to simply put these hashes inline where they always appear at.

Reply:
this is useful for the update script which you said was unneeded. i'll remove. how would you recommend updating?